owasp-suppressions.xml

```diff
@@ -2,18 +2,9 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress>
         <notes><![CDATA[
-   This is a false alert.
```

Comment:
Not sure why we changed the blanket suppression to specific ones - they're much harder to maintain. @laxmanchekka ?

Reply:
I was little over cautious about wild card exclusions. May not be required in this case